enigma-swing/src/main/java/cuchaz/enigma/gui/Gui.java

```diff
@@ -114,6 +113,16 @@
 	private final JTabbedPane openFiles;
 	private final HashBiMap<ClassEntry, EditorPanel> editors = HashBiMap.create();
 
+	private final WorkspaceRPanelContainer workspace = new WorkspaceRPanelContainer();
+
+	private final RPanel structureRPanel;
```

Comment:
Can this just be named `structurePanel`?

Reply:
There's also the actual structure panel in a field called `structurePanel`, so probably not.

Reply:
yup, I just picked the next best name I could think of, one of these (or both) will be renamed because all the other RPanels are just named *Panel

Reply:
there, now it's just `structurePanel` ;)